### ISSUE TITLE
No checkpoint file for DND Denoising?

### ISSUE THREAD

**Issue by AIisCool** [google-research/maxim#2]
Both links go to the SIDD checkpoint file.

**Comment by vztu** [comment]
Hi, DND dataset has provided no training set. The setting is we trained only on SIDD training set and evaluated on SIDD and DND test set.
